server/service/src/sync/translations/vvm_status_log.rs

```diff
@@ -0,0 +1,156 @@
+use crate::sync::{
+    sync_serde::empty_str_as_option_string,
+    translations::{
+        invoice_line::InvoiceLineTranslation, stock_line::StockLineTranslation,
+        store::StoreTranslation, user::UserTranslation, vvm_status::VVMStatusTranslation,
+    },
+};
+use anyhow::Error;
+use chrono::NaiveDateTime;
+use repository::{
+    vvm_status::vvm_status_log_row::{VVMStatusLogRow, VVMStatusLogRowRepository},
+    ChangelogRow, ChangelogTableName, StorageConnection, SyncBufferRow,
+};
+use serde::{Deserialize, Serialize};
+
+use super::{PullTranslateResult, PushTranslateResult, SyncTranslation};
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Serialize)]
+pub struct LegacyVVMStatusLogRow {
+    #[serde(rename = "ID")]
+    pub id: String,
+    #[serde(rename = "status_ID")]
+    pub status_id: String,
+    #[serde(rename = "date")]
+    pub created_datetime: NaiveDateTime,
```

Comment:
This should be seperate, you get date and time from OG, cause it's two separate fields and concat it~ there's examples in `translation/invoice.rs` (;

Reply:
Gonna fix this on parent branch - `7353-VVM-status-log`